packages/backend/src/problems/problem.controller.ts

```diff
@@ -0,0 +1,20 @@
+import { Body, Controller, Post } from "@nestjs/common";
+import { Public } from "src/auth/guards/jwt-auth.guard";
+
+import { CreateProblemDto } from "./create-problem.dto";
+import { ProblemService } from "./problem.service";
+
+@Controller("problems")
+export class ProblemController {
+    constructor(private readonly problemService: ProblemService) {}
+
+    @Public() // TODO: must be restricted to contributors
+    @Post("new")
+    async createProblem(@Body() dto: CreateProblemDto) {
+        const problem = await this.problemService.createProblem(dto);
+
+        console.log(JSON.stringify(problem));
```

Comment:
we should avoid console.log - if there is some sensible information to be logged (for example that user A created problem B, we could use a [logger](https://docs.nestjs.com/techniques/logger).